mne/beamformer/_lcmv.py

```diff
@@ -338,6 +338,100 @@ def apply_lcmv_epochs(epochs, filters, *, return_generator=False,
     return stcs
 
 
+@verbose
+def apply_lcmv_evoked_tfr(evoked_tfr, filters, verbose=None):
```

Comment:
I would not call this "evoked TFR", if this means the Hilbert beamformer. My mind goes automatically to Morlet and Wavelet. If this assumes bandpass-filtered or Hilbertized data, we should call it that.

Reply:
This is where the above discussion has consequences in the API; the bandpass-filter Hilbert-transformed data should really be stored in an object so that it's not a bunch of unruly lists. The object that makes sense already exists, it just hasn't been associated with this use yet.